local_init.bash

```diff
@@ -1,14 +1,19 @@
 #!/bin/bash
+
+set -xe
+
 CONTAINER_ID=$(docker container ls | grep enigmampc/client | cut -d' ' -f1)
-mkdir -p ~/.enigma
-mkdir -p ../build
-docker cp $CONTAINER_ID:/root/build/contracts ../build
-sed -e 's_http://contract_http://localhost_g;s_http://bootstrap_http://localhost_g' ../build/contracts/addresses.json > ../build/contracts/addresses.new
-mv ../build/contracts/addresses.new ../build/contracts/addresses.json
+
+rm -rf ../build ~/.enigma /tmp/enigma
+mkdir -p ../build /tmp/enigma
```

Comment:
In your spirit of keeping projects self-contained in their folder, I recommend you do not create `/tmp/enigma` but instead create `../tmp` like `../build`. Otherwise please make the argument why `/tmp/enigma` is a better choice

Reply:
Just because it's a place for temporary storage of data, nothing much to it.